samples/snippets/set_agent.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START dialogflow_set_agent_sample]
+
+from google.cloud.dialogflow_v2 import Agent
+from google.cloud.dialogflow_v2 import AgentsClient
+
+
+def set_agent(project_id, display_name):
+
+    parent = "projects/" + project_id
```

Comment:
Maybe we should use that helper:
https://github.com/googleapis/python-dialogflow/blob/b4986d43651ff373cc055f00c3c661773a681bb2/google/cloud/dialogflow_v2/services/agents/client.py#L209